src/components/proposals/brand/navbar/navbar.twig

```diff
@@ -0,0 +1,116 @@
+                    <path class="st0" d="M122.6 18.8h-6.4V0h38v18.9H147l6.5 43.4L167 0h19.2l14.4 62.3 5.2-43.4h-6.6V0h37.5v18.9h-6.2l-11.3 71.4h-30.6l-11.8-53.2 -12.1 53.1h-29.4L122.6 18.8z" />
+                    <path class="st0" d="M230.1 71.2h6.9L250.7 0h41l13.5 71.2h6.4v19H281l-2.9-22h-15.2l-2.7 22h-30L230.1 71.2 230.1 71.2zM276.5 51.7l-5.8-36.4 -6 36.4H276.5z" />
+                    <image src="../../images/uiowa-primary.png">
+                        <title id="logo-header">University of Iowa</title>
+                    </image>
+                </svg>
+            </a>
+        </div>
+
+
+      <div class="wrap-links">
+
+        <div class="top">
+            <div class="site-name-wrapper">
+              <h1 class="site-name" id="sections-heading">
+                <span></span>
+              <a>Employee Self-Service</a>
+              </h1>
+            </div>
+
+            <div class="toggle">
+              <a class="icon" role="button" title="menu" aria-expanded="false" href="javascript:void(0);" onclick="uidsUtils.navbar.menuToggle(this)">
+                <i class="fa fa-bars"></i>
+              </a>
+            </div>
+        </div>
+
+        <div class="site-links" id="myTopnav">
+
+          <div class="links">
+
+            <div class="dropdown">
+              <a class="dropbtn" href="javascript:void(0);" onclick="uidsUtils.navbar.dropDown(this, 'target1')">
+                  Drop Down 1
+                <span class="icons">
+                  <i class="fas fa-chevron-down"></i>
+                </span>
+              </a>
+
+              <div class="dropdown-content" id="target1">
+                  <a href="#">Link 1</a><span></span>
+                  <a href="#">Link Example 2 Link Example 2 Link Example 2</a><span></span>
+                  <a href="#">Link 3</a><span></span>
+              </div>
+
+            </div>
+
+            <a href="https://uiowa.edu/" title="title">
+             Link 1 longer title
+            </a>
+            <a href="https://uiowa.edu/" title="title">
+               Link 2
+            </a>
+            <a href="https://uiowa.edu/" title="title">
+              Link 3
+            </a>
+
+            <div class="dropdown">
+              <a class="dropbtn" href="javascript:void(0);" onclick="uidsUtils.navbar.dropDown(this, 'target2')">
+
+                <span class="">
+                  <i class="fa fa-user-alt"></i>
+                  Drop Down 2
+                </span>
+                <span class="icons">
+                  <i class="fas fa-chevron-down "></i>
+                </span>
+              </a>
+
+              <div class="dropdown-content" id="target2">
+                  <a href="#">Link 1</a><span></span>
+                  <a href="#">Link Example 2</a><span></span>
+                  <a href="#">Link 3</a><span></span>
+              </div>
```

Comment:
Ideally, this would not have demo content directly in the template. This could instead be set in `navbar.config.yml`